modules/tiled-gallery/tiled-gallery/tiled-gallery.js

```diff
@@ -138,29 +149,34 @@
 	}
 
 	function attachPlainResize( tiledGalleries ) {
-		$( window ).resize( function () {
+		window.addEventListener( 'resize', function () {
 			tiledGalleries.resizeAll();
 		} );
 	}
 
 	/**
 	 * Ready, set...
 	 */
-
-	$( document ).ready( function () {
+	function ready( fn ) {
+		if ( document.readyState !== 'loading' ) {
+			fn();
+		} else {
+			document.addEventListener( 'DOMContentLoaded', fn );
+		}
+	}
+	ready( function () {
 		var tiledGalleries = new TiledGalleryCollection();
 
-		$( 'body' ).on( 'post-load', function ( e, maybeResize ) {
-			if ( 'string' === typeof maybeResize && 'resize' === maybeResize ) {
-				tiledGalleries.resizeAll();
-			} else {
-				tiledGalleries.findAndSetupNewGalleries();
-			}
-		} );
-		$( document ).on( 'page-rendered.wpcom-newdash', function () {
+		document.body.addEventListener( 'is.post-load', function () {
 			tiledGalleries.findAndSetupNewGalleries();
 		} );
 
+		if ( jQuery ) {
+			jQuery( document ).on( 'page-rendered.wpcom-newdash', function () {
```

Comment:
**Note:** This is probably only used on `wpcom` and I suspect it's a jQuery event, so I've kept the `jQuery` check in.